nautobot_ssot_infoblox/diffsync/client.py

```diff
@@ -26,8 +41,13 @@ def __init__(
         wapi_version=PLUGIN_CFG.get("NAUTOBOT_INFOBLOX_WAPI_VERSION"),
         cookie=None,
     ):  # pylint: disable=too-many-arguments
-        """Initialization of infoblox class."""
-        self.url = url.rstrip()
+        """Initialize Infoblox class."""
+        parsed_url = urlparse(url.strip())
+        self.url = (
+            parsed_url._replace(scheme="https").geturl()
+            if parsed_url.scheme or not parsed_url.scheme
```

Comment:
I think we can just check if `parsed_url.scheme != "https"` as our function above accounts for no scheme and it makes it more explicit.

Reply:
Done.